src/native/external/zlib-ng.cmake

```diff
@@ -15,12 +15,14 @@ add_compile_options($<$<COMPILE_LANG_AND_ID:C,MSVC>:/wd4127>) # warning C4127: c
 add_compile_options($<$<COMPILE_LANG_AND_ID:C,MSVC>:/wd4242>) # 'function': conversion from 'unsigned int' to 'Pos', possible loss of data, in various deflate_*.c files
 add_compile_options($<$<COMPILE_LANG_AND_ID:C,MSVC>:/wd4244>) # 'function': conversion from 'unsigned int' to 'Pos', possible loss of data, in various deflate_*.c files
 
+# NOTE: Wasm is temporarily using system zlib, but will be switched to zlib-ng eventually. This branch should not affect the build.
```

Comment:
I added the comments in this file without commenting or removing the lines that affect wasm because we are now supposed to never import zlib-ng for wasm.